File: frontend/admin-panel/src/types/room/RoomUpdateDto.ts
export interface RoomUpdateDto {
    name?: string;
    floor?: number;
    roomTypeId?: number;
    description?: string;
    buildingId?: number;
    nodeId?: string;
    qrCode?: string;
    roomPolygon?: string;
    customColor?: string;
    customIconId?: number;
}

File: pstu_map/src/main/java/com/project/pstu_map/dto/room/RoomUpdateDto.java
package com.project.pstu_map.dto.room;

import lombok.Data;

@Data
public class RoomUpdateDto {
    private String name;
    private int floor;
    private Integer roomTypeId;
    private String description;
    private Integer buildingId;
    private String nodeId;
    private String qrCode;
    private String roomPolygon;
    private String customColor;
    private Integer customIconId;
}